test/src/node.rs

```diff
@@ -49,163 +37,196 @@ impl Drop for ProcessGuard {
     }
 }
 
+pub struct Node {
+    working_dir: String,
```

Comment:
please use `PathBuf` here since we refactored it in https://github.com/nervosnetwork/ckb/pull/2278